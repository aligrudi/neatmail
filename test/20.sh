# neatmail test
./mail mk -0 from:subject: $1
