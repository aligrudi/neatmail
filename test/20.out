N00000	[A <a@a.eu>]	[Message from A]
N00001	[B <b@b.eu>]	[Message from B]
N00002	[C <c@c.eu>]	[Message from C]
